src/gui/FxMixerView.cpp

```diff
@@ -395,6 +395,40 @@ void FxMixerView::deleteChannel(int index)
 
 
 
+void FxMixerView::deleteUnusedChannels()
+{
+	TrackContainer::TrackList tracks;
+	tracks += Engine::getSong()->tracks();
+	tracks += Engine::getBBTrackContainer()->tracks();
+
+	// go through all FX Channels
+	for(int i = m_fxChannelViews.size()-1; i>0; --i)
```

Comment:
On 01/04/2015 05:38 PM, Daniel Winzen wrote:
>
> Ok, adjusted. But for some reason this isn't consistently used
> everywhere in the code. Sometimes there are spaces and sometimes not.
>

That's why we have this

https://github.com/LMMS/lmms/wiki/Coding-conventions